openairclim/main.py

```diff
@@ -189,15 +189,44 @@ def run(file_name):
             ds_cont = oac.open_netcdf_from_config(
                 config, "responses", ["cont"], "resp"
             )["cont"]
-
             # Calculate Contrail Flight Distance Density (CFDD)
-            cfdd_dict = oac.calc_cfdd(config, inv_dict, ds_cont)
-
+            cfdd_dict = oac.calc_cfdd(
+                config, inv_dict, ds_cont
+            )
             # Calculate contrail cirrus coverage (cccov)
-            cccov_dict = oac.calc_cccov(config, cfdd_dict, ds_cont)
+            cccov_dict = oac.calc_cccov(
+                config, cfdd_dict, ds_cont
+            )
+
+            # if the results compared base inventory are required
```

Comment:
Here I do not understand the comment.

Reply:
It should say "if the results compared to base inventory are required", but I agree that that is not clear. What I want to say is that this if statement is processed if there is a base inventory to compare the input inventory to. I will quickly commit a better comment.